legacy/plugin.ts

```diff
@@ -44,6 +44,8 @@ export function adaptSingleProjectPlugin(plugin: SingleSubprojectPlugin): Plugin
 export interface BaseInspectOptions {
   // Include dev dependencies
   dev?: boolean;
+  // Skip any deps we could not resolve, and continue
+  skipUnresolved?: boolean;
```

Comment:
this will these come through as arg?
we could firm up the interface for that, instead of string[]

```
type Args = "dev" | "skip-unresolved" | "file" | "org" | "project-name"; // etc
interface BaseInspectOptions {
  args?: Args[];
}
```